src/posix/README.md

```diff
@@ -77,19 +77,25 @@ To build and program the device with RCP application, complete the following ste
 
    ```sh
    rm -rf build
-   script/build nrf52840 USB_trans -DOT_BOOTLOADER=USB -DOT_THREAD_VERSION=1.2
+   script/build nrf52840 USB_trans -DOT_BOOTLOADER=USB -DOT_THREAD_VERSION=1.3
```

Comment:
Can we simply remove the `-DOT_THREAD_VERSION=1.3`?

Reply:
+1 on this, using default version may be better.

Reply:
Makes sense.